### PR TITLE
#1509 After installing, Uninstall button in details is visible but gr…

### PR DESCRIPTION
**Pull request by MikhailArkhipov**
…eyed out

### CODE REVIEW

File: src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs [hunk 1]
```diff
@@ -157,7 +160,10 @@ internal class RPackageManagerViewModel : BindableBase, IRPackageManagerViewMode
             }
 
             package.IsChanging = true;
-            DispatchOnMainThread(() => InstallAsync(package));
+            DispatchOnMainThread(async () => {
+                await InstallAsync(package);
+                package.IsChanging = false;
```
Comment (AlexanderSher) [inline src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs]: Let's move this directly inside `InstallAsync`

File: src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs [hunk 2]
```diff
@@ -182,7 +188,10 @@ internal class RPackageManagerViewModel : BindableBase, IRPackageManagerViewMode
             }
 
             package.IsChanging = true;
-            DispatchOnMainThread(() => UninstallAsync(package));
+            DispatchOnMainThread(async () => {
+                await UninstallAsync(package);
+                package.IsChanging = false;
```
Comment (AlexanderSher) [inline src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs]: Same here - let's move it to the end of `UninstallAsync`

File: src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs (hunk 1 above)
Comment (MikhailArkhipov) [inline src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs]: OK. Install/Uninstall have multiple exit points though due to the prompt

Comment (MikhailArkhipov) [inline src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs]: Done

**Comment by AlexanderSher** [comment]
:+1:

**Pull request merged**